src/app/redux/form/formActions.js

```diff
@@ -32,6 +32,23 @@ const declareNewDevice = (formData) => {
   }
 }
 
+const editDeviceDetail = (formData) => {
+  const uri = process.env.REACT_APP_BASE_API_URL + '/devices/' + formData.id;
+  return (dispatch) => {
+    fetch(uri, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(formData)
+    }).then(() => {
+      dispatch(setFormPrompt('Successful!'));
```

Comment:
Don't use hard-code here

Reply:
Noted, I will pay more attention to these in the future